tools/parse_for_detail.py

```diff
@@ -12,11 +12,16 @@
 import os
 from pprint import pprint
 import re
+from pathlib import Path
+import logging
 
+PATTERNS = {'one' : r'\s*(?P<speakers>.*)\s*:\s*(?P<title>.*)\s*\|\s*(?P<event>.*)',
+            'two' : r'\s*(?P<speakers>.*)\s*-\s*(?P<title>.*)\s*-\s*(?P<event>.*)' }
 
-PAT = re.compile(r'^(.*)')
-SOURCE_KEY = 'description'
-CATEGORY = 'writethedocs-na-2016'
+PYVIDEO_DATA = Path(os.environ['PYVIDEO_DATA'])
+PAT = { key : re.compile(pattern) for key, pattern in PATTERNS.items() }
+SOURCE_KEY = 'title'
+CATEGORY = 'pydata-london-2019'
```

Comment:
Can you put the changes in this file in a separate PR, or revert them? Seems mostly specific to this event.